documentation/src/main/asciidoc/migration/index.adoc

```diff
@@ -43,9 +43,13 @@ Hibernate Search's requirements did not change in version {hibernateSearchVersio
 [[data-format]]
 == Data format and schema changes
 
-Indexes created with Hibernate Search {hibernateSearchPreviousStableVersionShort}
+Elasticsearch indexes created with Hibernate Search {hibernateSearchPreviousStableVersionShort}
 can be read from and written to with Hibernate Search {hibernateSearchVersion}.
 
+Reading and writing to Lucene indexes created with Hibernate Search {hibernateSearchPreviousStableVersionShort}
+using Hibernate Search {hibernateSearchVersion} may lead to exceptions, since there were incompatible changes applied to internal fields.
+You must recreate your Lucene indexes and reindex your database. The easiest way to do so is to use link:{hibernateSearchDocUrl}#indexing-massindexer[the `MassIndexer`] with link:{hibernateSearchDocUrl}#indexing-massindexer-parameters-drop-and-create-schema[`dropAndCreateSchemaOnStart(true)`].
+
```

Comment:
This should not be added. We're not introducing a backwards-incompatible change.